.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "federation-v2"]
```

Comment:
I think we also need to check-in (add) the `federation-v2` directory, after having checked out the specific commit we're interested in there (i.e. the `v0.0.4` tag), otherwise this will checkout the current `federation-v2` master